jest.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
     transform: {
         '^.+\\.jsx?$': 'babel-jest',
         '^.+/es/^.+$': 'babel-jest',
+        '^.+\\.ts?$': 'ts-jest',
```

Comment:
@niloo-fs I think we should also add `tsx` here, Same for `packages/indicators/jest.config.js` 🤔